### PR TITLE
Test for ORY evaluation installation

### PR DESCRIPTION
**Pull request by dariusztutaj**
### Description
 - implement test for ORY evaluation installation

### Links:
Resolves: #638
Based on: #637

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -70,7 +70,12 @@ test-all: test
 .PHONY: test-ory
 test-ory: export ORY_RECONCILER_INTEGRATION_TESTS = 1
 test-ory:
-	go test -v -timeout 5m ./pkg/reconciler/instances/ory/test
+	go test -v -timeout 5m ./pkg/reconciler/instances/ory/test/ory_integration_test.go
+
+.PHONY: test-ory-eval
+test-ory-eval: export ORY_RECONCILER_INTEGRATION_TESTS_EVALUATION = 1
```
Comment (tobiscr) [inline Makefile:76]: Do we really one more env-var to introduce? I'm not even happy that we have already two of them.

Comment (dariusztutaj) [reply Makefile]: You're right. Removed

File: Makefile [hunk 2]
```diff
@@ -70,7 +70,12 @@ test-all: test
 .PHONY: test-ory
 test-ory: export ORY_RECONCILER_INTEGRATION_TESTS = 1
 test-ory:
-	go test -v -timeout 5m ./pkg/reconciler/instances/ory/test
+	go test -v -timeout 5m ./pkg/reconciler/instances/ory/test/ory_integration_test.go
+
+.PHONY: test-ory-eval
+test-ory-eval: export ORY_RECONCILER_INTEGRATION_TESTS = 1
+test-ory-eval:
+	go test -v -timeout 5m ./pkg/reconciler/instances/ory/test/evaluation/ory_integration_evaluation_test.go
```
Comment (cnvergence) [inline Makefile:78]: We can also make use of EXECUTION_PROFILE env variable in the prow jobs, so we won't need to create separate tests. I will be happy to discuss it tomorrow.

**Comment by dariusztutaj** [comment]
Closing. It will be implemented in different way.

**Pull request closed without merging**